packages/circuit-ui/components/TopNavigation/components/UtilityLinks/UtilityLinks.tsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MouseEvent, KeyboardEvent, FC, SVGProps, HTMLProps } from 'react';
+import { css } from '@emotion/core';
+import { Theme } from '@sumup/design-tokens';
+import { Dispatch as TrackingProps } from '@sumup/collector';
+
+import styled, { StyleProps } from '../../../../styles/styled';
+import { hideVisually, navigationItem } from '../../../../styles/style-mixins';
+import { useClickEvent } from '../../../../hooks/useClickEvent';
+import Body from '../../../Body';
+
+const anchorStyles = ({ theme }: StyleProps) => css`
+  text-decoration: none;
+  padding: ${theme.spacings.kilo};
+  border-left: ${theme.borderWidth.kilo} solid ${theme.colors.n200};
+
+  ${theme.mq.kilo} {
+    padding: ${theme.spacings.kilo} ${theme.spacings.mega};
+  }
+`;
+
+const UtilityAnchor = styled.a(navigationItem, anchorStyles);
+
+const iconStyles = (theme: Theme) => css`
+  flex-shrink: 0;
+  width: ${theme.iconSizes.mega};
+  height: ${theme.iconSizes.mega};
+
+  ${theme.mq.kilo} {
+    margin-right: ${theme.spacings.byte};
+  }
+`;
+
+const labelStyles = ({ theme }: StyleProps) => css`
+  ${theme.mq.untilKilo} {
+    ${hideVisually()};
+  }
+`;
+
+const UtilityLabel = styled(Body)(labelStyles);
+
+export interface UtilityLinkProps extends HTMLProps<HTMLAnchorElement> {
+  /**
+   * Display an icon in addition to the text to help to identify the link.
+   * On narrow viewports, only the icon is displayed.
+   */
+  icon: FC<SVGProps<SVGSVGElement>>;
```

Comment:
Is there a reason why the logo props are not the same as the utility links props? I like these better, could the logo also not just be an object with an svg, a label, an href and an onClick instead of a custom ReactNode?

Reply:
The logo might need to be wrapped in an anchor or have additional elements (like the page title in the dashboard), so I prefer to keep it more open.

Reply:
Right, didn't think about the page title. Makes sense 👍